lib/instrument/index.js

```diff
@@ -17,7 +17,7 @@ const CHANNEL_OFFSET = 2 ** 32;
 const INSTRUMENT_SERVICE_NAME_VERSION_14 = 'com.apple.instruments.remoteserver.DVTSecureSocketProxy';
 const INSTRUMENT_SERVICE_NAME = 'com.apple.instruments.remoteserver';
 
-export const SERVICE = Object.freeze({
+export const CHANNEL = Object.freeze({
```

Comment:
would this be a breaking change? Why cannot we just add INSTRUMENT_CHANNEL to the list of other channels?

Reply:
No, only test code in this package refers to this for now.

Just thought `instrument` was redundant as the original definition's intention. Let me update.

Reply:
ah, if someone referred to this value over `appium-ios-device/build/lib/instrument`, could be a breaking change. Our modules do not have any references though.